extension/src/test/suite/experiments/index.test.ts

```diff
@@ -824,7 +824,7 @@ suite('Experiments Test Suite', () => {
         'experiments have been starred'
       ).to.be.true
     })
-  })
+  }).timeout(WEBVIEW_TEST_TIMEOUT)
```

Comment:
[F] All of these tests open a webview which means that they need the extended timeout.